.github/workflows/build-rails-base.yml

```diff
@@ -14,6 +14,9 @@ jobs:
           - ruby: '3.2.2'
             folder: '3.1.x' # slim bullseye
             tag: '3.2.2-slim@sha256:81743b016046dd7e6fe55b3b408ac37735786c9dff409ca2191daffa0a66fa7e'
+          - ruby: '3.2.3'
+            folder: '3.1.x' # slim bookworm
```

Comment:
If we use `3.1.x/Dockerfile` for other versions, should we rename the folder as `3.x.x`...?

Reply:
I thought about that, but I didn't want to add more change than strictly necessary here, and we're already using it for other 3.2.x images.